app/helpers/forms_helper.rb

```diff
@@ -38,57 +38,77 @@ def panel_block(**args, &block)
     end
   end
 
-  # Bootstrap checkbox: form, field, (label) text, class,
+  # <%= submit_button(form: f, button: button.t, center: true) %>
+  def submit_button(**args)
+    unless args[:form].is_a?(ActionView::Helpers::FormBuilder)
+      return args[:button]
+    end
+
+    opts = args.except(:form, :button, :class, :center)
+    opts[:class] = "btn btn-default"
+    opts[:class] += " center-block my-3" if args[:center] == true
+    opts[:class] += " #{args[:class]}" if args[:class].present?
+
+    args[:form].submit(args[:button], opts)
+  end
+
```

Comment:
@JoeCohen - Here's an example. This has been applied to every submit button on Earth. 

The main comment for this PR shows the difference in syntax. 

I'm currently writing helpers for all the other input types on Earth, and applying them in the forms. An eventual payoff from converting the forms to `form_with`, but whew it's a load of shoveling. Helpers are still evolving to accommodate the different situations/args i'm encountering.